spec/services/delivery_service_spec.rb

```diff
@@ -11,6 +11,7 @@
 
     it do
       expect(service.call).to eq(true)
```

Comment:
expect { service.call }.to change { order.reload.delivery_id }.from(nil).to(delivery.id)